dsn.go

```diff
@@ -99,6 +99,8 @@ type Config struct {
 	ClientStoreTemporaryCredential ConfigBool // When true the ID token is cached in the credential manager. True by default in Windows/OSX. False for Linux.
 
 	DisableQueryContextCache bool // Should HTAP query context cache be disabled
+
+	IncludeRetryReason bool // Should retried request contain retry reason
```

Comment:
should we add here information about the default value?

Reply:
but should we document it somewhere?

Reply:
I believe that no. This is actually more like a hidden feature, for emergency use cases. I don't think that users should switch it on their own.